### ISSUE TITLE
double space

### ISSUE THREAD

**Issue by larshp** [larshp/abaplint#413]
should report double space for

```abap
IF NOT  me->is_class_pool( me->program_name ) EQ abap_true.
```

`IF NOT  me->is_class_pool( me->program_name ) EQ abap_true.`
